src/FESpaces/Assemblers.jl

```diff
@@ -80,21 +83,28 @@ function assemble!(r,a::Assembler,cv::CellMatrix)
 end
 
 """
-Assembler that produces SparseMatrices from the SparseArrays package
+Assembler that produces SparseMatrices from the SparseArrays and SparseMatricesCSR packages
 """
-struct SparseMatrixAssembler{E} <: Assembler{SparseMatrixCSC{E,Int},Vector{E}}
+struct SparseMatrixAssembler{E,M} <: Assembler{AbstractSparseMatrix{E,Int},Vector{E}}
```

Comment:
`struct SparseMatrixAssembler{E,M} <: Assembler{M,Vector{E}}`

should be enough. The inner constructor takes care that M is <:AbstractSparseMatrix

Reply:
solved in 53d940e